driver-core/src/test/java/com/datastax/driver/core/DirectedGraphTest.java

```diff
@@ -58,24 +68,78 @@ public void should_sort_complex_graph() {
         g.addEdge("D", "B");
         g.addEdge("B", "A");
 
-        // Topological sort order should be : GH,FE,D,CB,A
+        // Topological sort order should be : GH,E,F,D,BC,A
         // There's no guarantee on the order within the same level, so we use sublists:
```

Comment:
You are right, there *is* a guarantee that it is ordered the same way.   I should update the test to no longer check on sublists and validate it as a whole, will do that.